docker/server/src/main/arch/arm64/image-bootstrap.properties

```diff
@@ -0,0 +1,11 @@
+Configuration.rootFile=grpc-api.prop
+io.deephaven.configuration.PropertyInputStreamLoader.override=io.deephaven.configuration.PropertyInputStreamLoaderTraditional
+LoggerFactory.silenceOnProcessEnvironment=true
+workspace=.
+devroot=.
+stdout.toLogBuffer=true
+stderr.toLogBuffer=true
+
+jpy.jpyLib=/usr/local/lib/python3.7/dist-packages/jpy.cpython-37m-aarch64-linux-gnu.so
```

Comment:
Rather than duplicating this file just to set one property (of two we know we'll need) can we not template it? Either gradle work out the file necessary for each build context, or let some RUN make it out of env vars, or compute the paths based on the image's contents?